dotCMS/src/main/java/com/dotmarketing/util/Config.java

```diff
@@ -357,14 +354,45 @@ private static void readEnvironmentVariables() {
     }
 
 
-    private static String envKey(final String theKey) {
+    public static String envKey(final String theKey) {
 
-        String envKey = ENV_PREFIX + theKey.toUpperCase().replace(".", "_");
-        while (envKey.contains("__")) {
-            envKey = envKey.replace("__", "_");
+        if(!theKey.startsWith(ENV_PREFIX)) {
+            String envKey = ENV_PREFIX + theKey.toUpperCase().replace(".", "_").replace("-","_");
```

Comment:
why not replaceAll?

Reply:
replaceAll takes a regex, that broke other tests.

Reply:
use StringUtils from apache commons